src/stylesheets/conversation.less

```diff
@@ -15,7 +15,7 @@
         background-color: #F8F9FA;
         padding: 18px 18px 22px 18px;
         border-bottom: solid 1px #E6E6E6;
-        min-height: 158px;
+        min-height: 90px; // app icon height + intro padding
```

Comment:
Ideally we could use vars, but I don't know if we have those values for this situation!